config/default.js

```diff
@@ -57,10 +57,8 @@ module.exports = {
   serverHost: '127.0.0.1',
   serverPort: 4000,
 
-  // By default, sending stats to DataDog is enabled but the host setting
-  // must also be non-empty.
-  useDatadog: true,
-  // These are set with environment variables.
+  // These are set with environment variables. We do not use datadog anymore
+  // but statsd/influxdb/grafana.
   datadogHost: null,
```

Comment:
I know you mentioned this in the original comment, but why can we not rename these to something more correct? Is it just because we don't want to have to change the environment variable name as well?

Reply:
Theoretically we could rename those config variables and update the file that binds env vars to config vars, but then it would make it hard to remember about that so I prefer to keep everything named `datadog` for now.

I will file a bug to update the env vars (actually duplicate them with new names until we can remove the old ones). After that we should be able to update our code.